### PR TITLE
Improve documentation

### LINKED ISSUE #64: What does the hyperparameter "normalize" refer to in PPO?
PPO hyperparameter configurations often refer to `normalize` as a logical, e.g. 
https://github.com/DLR-RM/rl-baselines3-zoo/blob/8ea4f4a87afa548832ca17e575b351ec5928c1b0/hyperparams/ppo.yml#L44

It's not clear to me what configuration this particular hyperparameter refers to, (if anything?)  e.g. I see A2C tunes the `normalize_advantage` parameter, but that's not a hyperparameter for PPO.  PPO has a boolean to normalize_image, but don't think that's it either.  Is this controlling whether or not the env gets wrapped in vector normalize?  

(For context here -- I've found the zoo scripts here particularly handy for tuning even for my custom environments, thanks! but am struggling to reproduce some of the tuned results by passing the best hyper-parameters directly to fresh initializations of the RL algorithms.  Thanks for the amazing work you've done in developing stable-baselines and the zoo!)

### PR DESCRIPTION
**Pull request by cboettig**
and a few additional comments regarding hyperparameter defaults in general.  Comments/feedback welcome!

closes #64

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,10 +85,42 @@ python train.py --algo sac --env Pendulum-v0 --save-replay-buffer
 ```
 It will be automatically loaded if present when continuing training.
 
+## Default agent hyperparameters
+
+Note that the default hyperparameters used in the zoo when tuning or training are not always the same as the defaults provided in [stable-baselines3](https://stable-baselines3.readthedocs.io/en/master/modules/base.html).  Consult the latest source code to be sure of these settings. For example:
+
+- PPO tuning assumes a network architecture with `ortho_init = False` when tuning, though it is `True` by [default](https://stable-baselines3.readthedocs.io/en/master/modules/ppo.html#ppo-policies). 
+
+- PPO and A2C will use seperate architectures for policy and value networks, e.g. `net_arch=[dict(pi=[64, 64], vf=[64, 64])`, while the default is a shared network, `net_arch=[64, 64]`
```
Comment (araffin) [inline README.md:94]: where did you see that the default was a shared network?

File: README.md [hunk 2]
```diff
@@ -105,6 +122,22 @@ Distributed optimization using a shared database is also possible (see the corre
 python train.py --algo ppo --env MountainCar-v0 -optimize --study-name test --storage sqlite:///example.db
 ```
 
+### Default agent hyperparameters
+
+Note that the default hyperparameters used in the zoo when tuning are not always the same as the defaults provided in [stable-baselines3](https://stable-baselines3.readthedocs.io/en/master/modules/base.html). Consult the latest source code to be sure of these settings. For example:
+
+- PPO tuning assumes a network architecture with `ortho_init = False` when tuning, though it is `True` by [default](https://stable-baselines3.readthedocs.io/en/master/modules/ppo.html#ppo-policies).
+
+- PPO and A2C will use seperate architectures for policy and value networks, e.g. `net_arch=[dict(pi=[64, 64], vf=[64, 64])`, while the default is a shared network, `net_arch=[64, 64]`
```
Comment (araffin) [inline README.md:131]: where did you see that the default was a shared network?

File: README.md [hunk 3]
```diff
@@ -105,6 +122,22 @@ Distributed optimization using a shared database is also possible (see the corre
...
+
+- Non-epsodic rollout in TD3 and DDPG assumes `gradient_steps = train_freq` and so tunes only `train_freq`.  
+
+- `normalize: True` means that the training environment will be wrapped in a [VecNormalize](https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/common/vec_env/vec_normalize.py#L13) wrapper. [Normalization uses](https://github.com/DLR-RM/rl-baselines3-zoo/issues/64) the default parameters of `VecNormalize`, with the exception of `gamma` which is set to match that of the agent.  This can be [overridden](https://github.com/DLR-RM/rl-baselines3-zoo/blob/v0.10.0/hyperparams/sac.yml#L239) using the appropriate `hyperparameters/algo_name.yml`, e.g.
```
Comment (araffin) [inline README.md:135]: before, it was written that the env was always wrapped, what made you think of that?

**Review by araffin — COMMENTED** [review]
Thank you very much for the PR =)

Only small comments otherwise LGTM ;)

PS: please use the PR template next time ;)

File: README.md (hunk 2 above)
Comment (cboettig) [reply README.md:131]: Good question, I was basing that on 
https://github.com/DLR-RM/stable-baselines3/blob/944dfdafe496490cb6a7dc0e2198c576e4d2ba98/stable_baselines3/dqn/policies.py#L41-L42

but realize I may have been misreading how far this default is inherited or something and should have at least flagged this comment more as a supposition on my part than a firm assertion!  (As mentioned earlier, overall I've found it rather tricky to figure out what the defaults are, hence my attempt to document them more explicitly somewhere, though perhaps the README isn't the best place to state some of these.  All comments in the PR here are based on my notes from my poor attempts to read the source or experiments over the past month and may well be wrong, so I can't say how much I appreciate the chance to verify or correct these assumptions with you!)

File: README.md (hunk 3 above)
Comment (cboettig) [reply README.md:135]: I must have misunderstood your comment in https://github.com/DLR-RM/rl-baselines3-zoo/issues/64#issuecomment-755217536, I thought you stated there that the env is always wrapped for all agents.  Guess I am still a bit fuzzy on what the default behaviour is here for each RL algo if I don't state it explicitly in the hyperparameters yaml file...

**Comment by cboettig** [comment]
My pleasure, and thanks for the detailed comments.  Apologies for any mistakes above, really appreciate the opportunity to confirm these with the maintainers!  Will be sure to use the PR template next time too.  

Thanks again for the amazing work by yourself and your team. It would be much harder for myself and my students, as ecologists and non-specialists, to get started exploring RL applications to conservation and environmental issues without such a well documented and implemented library (and believe me we've tried many others!)

File: README.md (hunk 2 above)
Comment (araffin) [reply README.md:131]: oh i see, you looked at DQN (which has only one Q-network) whereas PPO/A2C have two networks (actor and critic)

>tricky to figure out what the defaults

we should probably make it explicit in SB3 doc. but for short: we use the network architectures from the original papers (so [256, 256] for SAC, separated [64, 64] networks for PPO, [400, 300] for TD3, ...) and same for the default hyperparameters.

**Comment by araffin** [comment]
>My pleasure, and thanks for the detailed comments. Apologies for any mistakes above, really appreciate the opportunity to confirm these with the maintainers! Will be sure to use the PR template next time too.

no worry, don't hesitate to open issues if you have questions about the behavior/defaults of the code.
In fact, this kind of feedback is valuable to us as we no longer have the necessary hindsight to see what is obvious and what is not ;)

>Thanks again for the amazing work by yourself and your team. It would be much harder for myself and my students, as ecologists and non-specialists, to get started exploring RL applications to conservation and environmental issues without such a well documented and implemented library (and believe me we've tried many others!)

thank you for your kind message =)

**Review by araffin — APPROVED** [review]
LGTM =)
